index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <nav id="navcontainer">
+    <div>
+      <h3 id="menulogo">John Doe</h3>
+    </div>
+    <div>
+      <img id="burgermenu" src="./img/burger.png" alt="">
```

Comment:
- [ ] Since we are/will be using this hamburger menu to navigate it's better to embed it in a `<nav></nav>` tag and use a header tag where you used a nav tag.